atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -213,7 +214,10 @@ def _get_remote_urls_helper(self, koji_build):
             elif archive['type_name'] == 'json' and archive['filename'] == 'remote-source.json':
                 cachito_json_url = os.path.join(remote_sources_path, archive['filename'])
 
-        return remote_sources_urls, {dest_name: cachito_json_url}
+        if cachito_json_url and dest_name:
```

Comment:
Can you explain me how dest_name works here? How is that aligned with json file?

Reply:
Unittest?

Reply:
Can actually be `cachito_json_url XOR dest_name == True` a valid case? Should we raise exception if only one is set?

Reply:
added unitests